parse.py

```diff
@@ -928,66 +1041,65 @@ def price_item(text):
                         price_vals = [[str(price_val) + price_curr]]
 
-                        print("[!] Not enough data to confidently price this item.")
+                        logging.info("[!] Not enough data to confidently price this item.")
                         if config.USE_GUI:
                             gui.show_price(price, price_vals, time, True)
                     else:
-                        print(f"[$] Price: {Fore.YELLOW}None \n\n")
-                        print("[!] Not enough data to confidently price this item.")
+                        logging.info(f"[$] Price: {Fore.YELLOW}None{Fore.RESET} \n\n")
+                        logging.info("[!] Not enough data to confidently price this item.")
                         if config.USE_GUI:
                             gui.show_not_enough_data()
 
             elif trade_info is not None:
-                print("[!] No results!")
+                logging.info("[!] No results!")
                 if config.USE_GUI:
                     gui.show_not_enough_data()
 
     except NotFoundException as e:
-        print("[!] No results!")
+        logging.info("[!] No results!")
         if config.USE_GUI:
             gui.show_not_enough_data()
 
     except InvalidAPIResponseException as e:
-        print(f"{Fore.RED}================== LOOKUP FAILED, PLEASE READ INSTRUCTIONS BELOW ==================")
-        print(
+        logging.info(f"{Fore.RED}================== LOOKUP FAILED, PLEASE READ INSTRUCTIONS BELOW =================={Fore.RESET}")
+        logging.info(
             f"[!] Failed to parse response from POE API. If this error occurs again please open an issue at {PROJECT_URL}issues with the info below"
         )
-        print(f"{Fore.GREEN}================== START ISSUE DATA ==================")
-        print(f"{Fore.GREEN}Title:")
-        print("Failed to query item from trade API.")
-        print(f"{Fore.GREEN}Body:")
-        print("Macro failed to lookup item from POE trade API. Here is the item in question.")
-        print("====== ITEM DATA=====")
-        print(f"{text}")
-        print(f"{Fore.GREEN}================== END ISSUE DATA ==================")
-        print(f"{Fore.RED}================== LOOKUP FAILED, PLEASE READ INSTRUCTIONS ABOVE ==================")
+        logging.info(f"{Fore.GREEN}================== START ISSUE DATA =================={Fore.RESET}")
+        logging.info(f"{Fore.GREEN}Title:{Fore.RESET}")
+        logging.info("Failed to query item from trade API.")
+        logging.info(f"{Fore.GREEN}Body:{Fore.RESET}")
+        logging.info("Macro failed to lookup item from POE trade API. Here is the item in question.")
+        logging.info("====== ITEM DATA=====")
+        logging.info(f"{text}")
+        logging.info(f"{Fore.GREEN}================== END ISSUE DATA =================={Fore.RESET}")
+        logging.info(f"{Fore.RED}================== LOOKUP FAILED, PLEASE READ INSTRUCTIONS ABOVE =================={Fore.RESET}")
```

Comment:
do we want to change this into a logging block with \n characters? it may look neater.

Reply:
Not concerned with this in these sets of commits. Something that should be taken up after the rewrite PR, or on upstream.